frontend/public/components/environment.jsx

```diff
@@ -186,7 +187,7 @@ export class EnvironmentPage extends PromiseComponent {
       const keyString = _.isArray(rawEnvData) ? rawEnvData[i].name : rawEnvData.from.name;
       return <div key={keyString} className="co-m-pane__body-group">
         { _.isArray(rawEnvData) && <h1 className="co-section-title">Container {keyString}</h1> }
-        <NameValueEditor nameValueId={i} nameValuePairs={envVar} updateParentData={this.updateEnvVars} addString="Add Value" nameString="Name" readOnly={readOnly}/>
+        <NameValueEditor nameValueId={i} nameValuePairs={envVar} updateParentData={this.updateEnvVars} addString="Add Value" nameString="Name" readOnly={readOnly} allowSorting={true}/>
```

Comment:
You need to wrap this component in an `AsyncComponent`, not `PairElement`. This component brings along `PairElement`, so just wrapping the top-level component using `import()` will fix the bundle size increase.